python/dronin/telemetry.py

```diff
@@ -672,6 +722,10 @@ def __init__(self, cmdline, shell=True, *args, **kwargs):
 
         import subprocess
 
+        if isinstance(cmdline, str):
```

Comment:
Before we had a shell inbetween the sim and us... If we send a signal to our child process, it takes a bit for the signal to be delivered to the sim and we risk having two instances running.  So we get rid of the intervening shell and tokenize the cmdline ourselves.

Simultaneously this is extended to allow passing in a pre-tokenized list, though we don't use that currently.